main.js

```diff
@@ -102,31 +111,158 @@ function createDetailPage(travelPackage) {
   return $div
 }
 
-//return to home
+//calculate total
+function calculateTotal(cartedTravelPackage) {
+  var total = 0
+  for (var i = 0; i < cartedTravelPackage.length; i++) {
+    total += cartedTravelPackage[i].price
+  }
+  return total
+}
+
+//display total
+function displayTotal() {
+  var $div = document.getElementById('viewtotal')
+  var $span = document.createElement('span')
+
+  $div.innerHTML = ""
```

Comment:
Display total should take arguments. Calculate the total and pass it into `displayTotal`.